### PR TITLE
Switch to new Microsoft.DotNet.SharedFramework.Sdk

### PR DESCRIPTION
**Pull request by jkoritzinsky**
Switch to the new unified tooling in the Microsoft.DotNet.SharedFramework.Sdk package.

### CODE REVIEW

**Comment by mmitche** [comment]
@jkoritzinsky Can you resolve the conflicts and figure out what still needs to be done to get WD onto the new SDK?

**Comment by jkoritzinsky** [comment]
All that's left is to resolve the conflicts (which are all from the Maestro auto-update PRs IIRC). Everything is working in this PR on the new SDK.

I'll resolve the conflicts when I'm back from vacation.

File: pkg/windowsdesktop/Directory.Build.props [hunk 1]
```diff
@@ -29,8 +25,8 @@
     -->
     <PermitDllAndExeFilesLackingFileVersion>true</PermitDllAndExeFilesLackingFileVersion>
 
-    <!-- Generate the runtimeconfig with the correct Microsoft.NETCore.App version. -->
-    <MicrosoftNETCoreAppRuntimeConfigVersion>$(MicrosoftNETCoreAppRuntimewinx64Version)</MicrosoftNETCoreAppRuntimeConfigVersion>
+    <!-- Workaround until https://github.com/dotnet/arcade/pull/6566 is merged and consumed. -->
```
Comment (RussKie) [inline pkg/windowsdesktop/Directory.Build.props:28]: This PR has been merged, is this still required?

File: pkg/Directory.Build.props [hunk 2]
```diff
@@ -12,207 +11,38 @@
...
-    <RIDPropsFile Condition="'$(RIDPropsFile)' == ''">$(MSBuildThisFileDirectory)\windowsdesktopRIDs.props</RIDPropsFile>
-  </PropertyGroup>
-
-  <Import Project="$(RIDPropsFile)" />
-
+  
+  <!--
+    SDK currently does not know about net6.0 TargetFramework.
+    We need to manually add these to correctly resolve the targeting packs.
+    https://github.com/dotnet/sdk/issues/13950
```
Comment (RussKie) [inline pkg/Directory.Build.props:28]: This appears to have been resolved.

File: NuGet.config [hunk 3]
```diff
@@ -13,6 +13,7 @@
     <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
     <add key="dotnet6-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6-transport/nuget/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="jkoritzinsky-test" value="https://www.myget.org/F/jkoritzinsky/api/v3/index.json" />
```
Comment (RussKie) [inline NuGet.config:16]: Is this still required?

**Review by RussKie — COMMENTED** [review]
Few questions

File: pkg/Directory.Build.props (hunk 2 above)
Comment (jkoritzinsky) [reply pkg/Directory.Build.props:28]: To remove these items, we need to update the global.json to reference a 6.0-based SDK.

**Comment by jkoritzinsky** [comment]
I've addressed all of the feedback and CI is green again. @RussKie can you take another look?

**Review by RussKie — COMMENTED** [review]
:shipit:

**Pull request merged**